### PR TITLE
A few minor fixes

### PR DESCRIPTION
**Pull request by hackergrrl**
Just a few things I noticed while working through the examples (#22 in particular).

### CODE REVIEW

File: examples/ipns/readme.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ are quite simple.
 First, you'll need some content to publish:
 
 ```
-$ echo "Let's have some mutable fun!" | ipfs add
```
Comment (whyrusleeping) [inline examples/ipns/readme.md:11]: but i like exclamation points!

**Comment by whyrusleeping** [comment]
LGTM

**Comment by hackergrrl** [comment]
Fine, you win. I wanted to avoid the awkwardness of having to escape ! with double quotes or the ' in "Let's" with single quotes. :)

**Comment by whyrusleeping** [comment]
@noffle haha, i guess thats a fair point. Thanks!

File: examples/api/service/readme.md [hunk 2]
```diff
@@ -28,8 +28,8 @@ Set up an ipfsnode.
 ```
 func main() {
 	// Basic ipfsnode setup
-	r := fsrepo.At("~/.ipfs")
-	if err := r.Open(); err != nil {
+	r, err := fsrepo.Open("~/.ipfs")
+	if err != nil {
```
Comment (jbenet) [inline examples/api/service/readme.md:32]: thanks! 

wonder if we ought to try and make our examples work as tests somehow. (i like how go does it-- "examples" are just code that is runnable, and that godoc treats specially.) this is probably unrealistic for anything complicated though...

**Comment by jbenet** [comment]
thanks @noffle!

**Pull request merged**